coins/monero/tests/send.rs

```diff
@@ -7,19 +7,19 @@ use lazy_static::lazy_static;
 use zeroize::Zeroizing;
 use rand_core::OsRng;
 
+use curve25519_dalek::{constants::ED25519_BASEPOINT_TABLE, scalar::Scalar};
+
 #[cfg(feature = "multisig")]
 use blake2::{digest::Update, Digest, Blake2b512};
-
-use curve25519_dalek::constants::ED25519_BASEPOINT_TABLE;
-
 #[cfg(feature = "multisig")]
-use dalek_ff_group::Scalar;
+use dalek_ff_group;
```

Comment:
This is pointless. You can just use dalek_ff_group::Scalar below without the use. If you're using members of dalek_ff_group regularly, I'd recommend `use dalek_ff_group as dfg;`, yet this here can just be removed.